### PR TITLE
Video Changes

### PR DESCRIPTION
**Pull request by kamranpirwani**
### Purpose
Make the relevant changes to get videos working for brightwheel

### Solution
Ensure we can pass in a video or photo mode to start the GalleryActivity with the relevant media showing up. Make any other changes for the library to respect videos and photos

### CODE REVIEW

File: app/src/test/java/com/tangxiaolv/telegramgallery/app/ExampleUnitTest.java [hunk 1]
```diff
@@ -1,15 +0,0 @@
-package com.tangxiaolv.telegramgallery.app;
```
Comment (kamranpirwani) [inline app/src/test/java/com/tangxiaolv/telegramgallery/app/ExampleUnitTest.java:1]: Was not building, just removed

**Review by GuillaumeDelente — COMMENTED** [review]
When no videos are available, message is "No photos yet"

File: app/src/test/java/com/tangxiaolv/telegramgallery/app/ExampleUnitTest.java (hunk 1 above)
Comment (GuillaumeDelente) [reply app/src/test/java/com/tangxiaolv/telegramgallery/app/ExampleUnitTest.java:1]: Dude you can't remove such a useful test!

**Comment by kamranpirwani** [comment]
@GuillaumeDelente Verified the:
"When no videos are available, message is "No photos yet" 
issue is fixed and see the correct message for photos and videos

**Review by GuillaumeDelente — APPROVED** [review]

**Pull request merged**